src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,314 @@
+Let's fill it out with:
+```yaml
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: API gateway caching example
+  description: API gateway caching example
+
+servers:
+  - url: https://{domain}:{port}
+    description: The local API server
+    variables:
+      domain:
+        default: localhost
+        description: api domain
+      port:
+        enum:
+          - '8092'
+        default: '8092'
+
+paths:
+  /healthcheck:
+    get:
+      operationId: healthcheck-operation
+      responses:
+        default:
+          description: example vert.x healthcheck
+
+  /product/id:
+    get:
+      operationId: product-api-caching-proxy-operation
+      responses:
+        default:
+          description: Target API (Product API) caching proxy.
+```
+This is whole structure of this file required in our case. In more detail it was explained in [this tutorial](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/) 
+The most important for us here is `/product/id` endpoint definition. As you can see we are connecting this endpoint with
+`product-api-caching-proxy-operation` which we've defined before. Moreover here we've provided some other specification 
+but it was described in other tutorials, so if you don't feel familiar with it, please check them out.
+
+We have all Knot.x specific files created, now we are gonna focus on project setup and deployment.  
+
+## Project configuration
+The first thing we need to do is to specify configuration files. Let's start in the project level directory and simply create `api-cache.yml`
+Let's specify the following configuration:
+``` yml
+version: '3.7'
+
+networks:
+  knotnet:
+
+services:
+
+  webapi:
+    image: rodolpheche/wiremock
+    volumes:
+      - "./common-services/webapi:/home/wiremock"
+    ports:
+      - "3000:8080"
+    networks:
+      - knotnet
+
+  knotx:
+    image: knotx-example/api-cache:latest
+    command: ["knotx", "run-knotx"]
+    ports:
+      - "8092:8092"
+      - "18092:18092"
+    networks:
+      - knotnet
+```
+
+First and foremost it specifies Docker virtual network in which our services will be deployed. Then we have some service configs.
+The first one is `webapi`
+- `image` - docker image is provided here, 
+- `volumes` - directories to which Docker will have access, in our case it's directory where all Wiremock configs are stored,
+- `ports` - on the left side is provided port accessible from `localhost`, and on the right side we have port number available in virtual environment 
+- `networks` - specifies in which docker network, service will be deployed.
+
+Then we have `knotx` service specification, it's very similar to previous one, so only one property is worth to explain 
+- `command` - this property is used to point Knot.x starting commands.
+
+Next step in our configuration is to properly set up `settings.gradle.kts` (also at the project level directory) 
```

Comment:
Overall comment to this part of the tutorial. I feel like there to much details here. Aren't those things explained in Getting Started With Docker?